### PR TITLE
EHN: remove deployement_dir and use absolute path for RAMP components

### PR DESCRIPTION
**Pull request by glemaitre**

### CODE REVIEW

File: ramp-database/ramp_database/testing.py [hunk 1]
```diff
@@ -47,18 +48,35 @@ def create_test_db(database_config, ramp_config):
...
     database_config = database_config['sqlalchemy']
     # we can automatically setup the database from the config file used for the
     # tests.
     ramp_config = generate_ramp_config(ramp_config)
-    shutil.rmtree(ramp_config['deployment_dir'], ignore_errors=True)
+
+    # FIXME: we are recreating the deployment directory but it should be
+    # replaced by an temporary creation of folder.
+    match = SequenceMatcher(
+        None, ramp_config['ramp_kit_dir'], ramp_config['ramp_submissions_dir']
```
Comment (jorisvandenbossche) [inline ramp-database/ramp_database/testing.py]: you can use `os.path.commonpath` here (eg between ramp_kit_dir and ramp_data_dir), that would already be much simpler I think. 
But we could also just hard code it, as for the tests we know what the it is (or which part of the path it is, eg do a os.path.split twice on ramp_kit_dir)

File: ramp-database/ramp_database/tests/data/ramp_config_boston_housing.yml [hunk 2]
```diff
@@ -3,12 +3,11 @@ ramp:
...
-    data_dir: ramp-data/boston_housing
-    kit_dir: ramp-kits/boston_housing
-    submissions_dir: submissions
-    predictions_dir: preds
-    logs_dir: log
+    data_dir: /tmp/databoard_test/ramp-data/boston_housing
+    kit_dir: /tmp/databoard_test/ramp-kits/boston_housing
+    submissions_dir: /tmp/databoard_test/submissions
+    predictions_dir: /tmp/databoard_test/preds
+    logs_dir: /tmp/databoard_test/log
```
Comment (jorisvandenbossche) [inline ramp-database/ramp_database/tests/data/ramp_config_boston_housing.yml]: question: if you have multiple events, are they all stored in the same directory? (it's not stored in a subdir of this logs_dir path?)

File: ramp-database/ramp_database/testing.py (hunk 1 above)
Comment (glemaitre) [reply ramp-database/ramp_database/testing.py]: oh. i did not search properly. Thanks

File: ramp-database/ramp_database/tests/data/ramp_config_boston_housing.yml (hunk 2 above)
Comment (glemaitre) [reply ramp-database/ramp_database/tests/data/ramp_config_boston_housing.yml]: This is used by the dispatcher. So if you have a config file by event, you can change prediction and log directories

**Pull request merged**